index.Rmd

```diff
@@ -30,6 +30,10 @@ a:visited {color: #91170a;}
 # Basic *R* knowledge useful for Geo-Scripting
 Scripting with *R* to handle spatial problems requires a core set of basic *R* skills. To prepare you well for the coming weeks, we've summarized what we think are important skills for geoscripting below.
 
+```{block type="alert alert-info"}
+**Note** that this tutorial uses the basic programming structure with lists, strings, and more. For a refresher on these elements click [here](https://geoscripting-wur.github.io/RPythonBasics/).
+```
+
```

Comment:
The link looks correct. I think you can even use relative links, i.e. `[here](../RPythonBasics/)`, though it needs testing to make sure that would work as expected.

Yes, good idea to add a bit more info on e.g. how to install packages in R, because we never really cover that. I'm not sure if we should necessarily focus on RStudio (RKWard is more stable on Linux), but we could cover the common elements.